iosApp/iosApp/Pages/TripDetails/TripDetailsHeader.swift

```diff
@@ -41,12 +45,15 @@ struct TripDetailsHeader: View {
             Text("Live")
                 .font(Typography.footnote)
         }
+        .accessibilityElement()
+        .accessibilityAddTraits(.isHeader)
+        .accessibilityHeading(.h1)
```

Comment:
question: Is h1 necessary here? My expectation from web headings is that there is [typically only one h1 on a page](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#avoid_using_multiple_h1_elements_on_one_page).

Reply:
Yup, I think this was a copy/paste error that grabbed too many lines.